openstack/nova/alerts/vmware-rebalancer/vmware-rebalancer.alerts

```diff
@@ -0,0 +1,17 @@
+groups:
+  - name: vmware-rebalancer.alerts
+    rules:
+      - alert: VmwareRebalancerLowPlacableVMsPerVcenter
+        expr: wm_vmware_rebalancer_placeable_vms_per_vcenter{has_bbs_with_custom_hana_exclusive_host="False",has_bbs_without_custom_hana_exclusive_host="True",flavor="g_c128_m512"} <= 3
+        for: 30m
+        labels:
+          severity: warning
+          service: compute
+          support_group: compute
+          tier: vmware
+          context: "vCenter Capacity {{ $labels.vcenter }}"
+          meta: "The vCenter `{{ $labels.vcenter }}` is (almost) full and only allows scheduling less than 4 VMs."
+          playbook: docs/support/playbook/nova/alerts/cc3test-alert-vcenter-full.md
```

Comment:
I followed naming convention of the other files, but am unsure why they have prefix `cc3test`.

- [ ] Also, this playbook does not exist yet.

Reply:
cc3test prefix comes from the cc3tests that we're running - our smoketests. Since this alert doesn't come from cc3test, it should have that prefix.